composer/lib/dependabot/composer/file_fetcher.rb

```diff
@@ -26,7 +26,7 @@ def self.required_files_message
       def ecosystem_versions
         {
           package_managers: {
-            "composer" => Helpers.composer_version(parsed_composer_json, parsed_lockfile) || "unknown"
+            "composer" => Helpers.composer_version(parsed_composer_json, parsed_lockfile)
```

Comment:
If you check the `composer_version` method you will see there is no case we are returning nil. Either we are returning V1 or V2 or an error is going to be raised. So the `unknown` is removed

https://github.com/dependabot/dependabot-core/pull/10732/files#diff-9291c98c3b39d1cbadbf64e8e64c1739e1b3fb0950b93c8b8c688328349307edL25